src/BuildScriptGenerator/Node/NodeLanguageDetector.cs

```diff
@@ -149,5 +154,51 @@ private string DetectNodeVersion(dynamic packageJson)
 
             return nodeVersion;
         }
+
+        private bool IsStaticSite(ISourceRepo sourceRepo)
+        {
+            return IsHugoSite(sourceRepo);
+        }
+
+        private bool IsHugoSite(ISourceRepo sourceRepo)
```

Comment:
Let's make this `static` and call from NodePlatform to avoid duplicating logic

Reply:
Moved this functionality into its own helper class so it's not coupled to the `NodeLanguageDetector`